middlewares/isUser.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-trailing-spaces */
+import models from '../models';
+
+const { User } = models;
+
+/**
+ * class
+*/
+export default class {
+  /**
+     * Check if a user exist in the database
+     * @constructor
+     * @param {object} req - The request object
+     * @param {Object} res - The response object
+     * @param {object} next - Method to call another middleware
+     */
+  static async getUser(req, res, next) {
```

Comment:
Is there a specific reason you abstracted this middleware to its own file as opposed to including it in the existing `userValidations` module?

Reply:
not really, just that I taught it was meant to validate user registrations alone. But I'm merge them together right away

Reply:
Done